git/README.md

```diff
@@ -11,6 +11,11 @@ Prefix commit message with a [JIRA](https://www.atlassian.com/software/jira) ID
 - JIRA-123 Add Git conventions
 - Add Git conventions (JIRA-123)
 
+## Commit messages
+- Commit messages should be clear and concise in describing the content of the commit
+- Commit messages should be written in English
```

Comment:
US English

Reply:
Changed! 👍

Reply:
You changed it on the gitflow page, not in the README 😶